testing/tests/DevExpress.exporter/jspdfParts/jspdfTestHelper.js

```diff
@@ -20,12 +18,21 @@ class JSPdfDataGridTestHelper {
     }
 
     checkCellsContent(headCellsArray, bodyCellsArray, actualAutoTableOptions) {
-        this._iterateCells(headCellsArray, (content, rowIndex, columnIndex) => {
-            assert.equal(actualAutoTableOptions.head[rowIndex][columnIndex].content, content, `AutoTable head[${rowIndex}][${columnIndex}].content`);
-        });
-        this._iterateCells(bodyCellsArray, (content, rowIndex, columnIndex) => {
-            assert.equal(actualAutoTableOptions.body[rowIndex][columnIndex].content, content, `AutoTable body[${rowIndex}][${columnIndex}].content`);
-        });
+        if(headCellsArray.length > 0) {
+            this._iterateCells(headCellsArray, (content, rowIndex, columnIndex) => {
+                assert.equal(actualAutoTableOptions.head[rowIndex][columnIndex].content, content, `AutoTable head[${rowIndex}][${columnIndex}].content`);
+            });
+        } else {
+            assert.equal(actualAutoTableOptions.head.length, 0, 'AutoTable head is empty');
```

Comment:
Try to use strictEqual & deepEqual for comparison of the object variables and not only, it's realy useful.

Reply:
Changed 'equal' to 'strictEqual' where necessary